test/quantization/fx/test_quantize_fx.py

```diff
@@ -669,9 +672,10 @@ def conv_res_relu_extra_inputs_getter(pattern):
             relu, (_, _, extra_input) = pattern
             return [extra_input]
 
-        conv_relu_config = BackendPatternConfig((nn.ReLU, nn.Conv2d)) \
+        conv_relu_config = BackendPatternConfig((nn.Conv2d, nn.ReLU)) \
             .set_fuser_method(fuse_conv_relu)
-        conv_res_relu_config = BackendPatternConfig((nn.ReLU, (torch.add, nn.Conv2d, MatchAllNode))) \
+        conv_res_relu_config = BackendPatternConfig() \
+            ._set_pattern_complex_format((nn.ReLU, (torch.add, nn.Conv2d, MatchAllNode))) \
```

Comment:
@andrewor14 simple pattern is using constructor args, but complex pattern is using `_set_pattern_complex_format`, this feels a bit inconsistent, can we change the API for simple pattern to use `set_pattern` instead?

Reply:
Yes after this PR there are actually two ways of setting the simple pattern, through the constructor (for BC), and through a new `set_pattern` that's analogous to `_set_pattern_complex_format`. I didn't want to just remove the constructor method because it would break all existing use cases